src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -143,6 +150,7 @@ export function AiAssistantSelectionPage() {
                     <EuiSpacer size="s" />
                     <EuiCallOut
                       iconType="warning"
+                      data-test-subj="pluginsAiAssistantSelectionPageSecurityDocumentationCallout"
```

Comment:
nit: Consider matching the width of the callout + content so the callout doesn't stretch the full parent card

<img width="987" alt="image" src="https://github.com/user-attachments/assets/4186dfc9-26e7-4479-9ddf-7f3505e7711f">

Reply:
<img width="2556" alt="Screenshot 2024-07-31 at 11 03 14" src="https://github.com/user-attachments/assets/e38850d5-40b4-49ea-89b8-4fce9e145070">